test_cases/test_persons.py

```diff
@@ -1,17 +1,17 @@
 import unittest
-from app.person import Staff, Fellow
+from ..app.person import Staff, Fellow
 
 class TestPerson(unittest.TestCase):
 	"""Test Person added"""
 	def setUp(self):
```

Comment:
Here I'm using the setUp()  method in Testcase, to prepare the test fixtures and avoid repeatedly, constructing class objects for each test